### PR TITLE
move apps to role format

### PR DESCRIPTION
**Pull request by PurpleNinja225**
**What this PR does / why we need it**:

Rolling move of apps to the role based format. will make 1 commit/app from here going fwd.

**Which issue (if any) this PR fixes**:

Fixes #

**Any other useful info**:

added app template role to show example and to make moving apps to roles a bit faster (for myself at least)

##Currently moved apps in this PR:
```yml
nzbget:
pyload:
tautulli:
get_iplayer:
jackett:
```

### CODE REVIEW

**Comment by davestephens** [comment]
Thanks! I'll try and get to this over the weekend. 👍

Going forward, it helps me with merging and review if you can keep PRs to
once piece of functionality at a time (this is actually in the contribution
docs 🙂)

**Comment by PurpleNinja225** [comment]
Will do. Just added jackett cause I already had it done before seeing this :sweat_smile:

File: nas.yml [hunk 1]
```diff
@@ -98,6 +98,11 @@
         - homeassistant
```
Comment (PurpleNinja225) [inline nas.yml:98]: Missing get_iplayer role

File: roles/application-template/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (PurpleNinja225) [inline roles/application-template/defaults/main.yml:1]: mv to its own PR

File: roles/application-template/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (PurpleNinja225) [inline roles/application-template/tasks/main.yml:1]: mv to its own PR

**Pull request merged**